src/onefetch/info.rs

```diff
@@ -28,35 +28,52 @@ pub struct Info {
     number_of_branches: usize,
     license: String,
     pub colors: Vec<Color>,
+    pub color_set: TextColor,
     pub config: Cli,
 }
 
+pub struct TextColor {
+    title: Color,
+    tilde: Color,
+    underline: Color,
+    subtitle: Color,
+    colon: Color,
+    info: Color,
+}
```

Comment:
In order to prevent the info.rs file from getting any bigger, it may be a good idea to move TextColor struct + get_color_set() into a separate file.